kensu/utils/kensu.py

```diff
@@ -198,6 +215,10 @@ def register_schema_name(self, ds, schema):
         if "in-mem" in name and ds.format is not None:
             name = name + " of format=" + str(ds.format or '?')
         self.schema_name_by_guid[schema.to_guid()] = name
+        try:
+            self.logical_name_by_guid[schema.to_guid()] = ds.categories[0].split("::")[1]
```

Comment:
in some cases taking only part of ds category could yield non-distinct names, isn't it? why don't we take full category/logical name?
otherwise the variable may need to be updated to like `logical_name_by_guid`?